webapi/webapi-nfc-w3c-tests/nfc/support/nfc_common.js

```diff
@@ -29,7 +29,12 @@ Authors:
 
 */
 
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharness.js\"><\/script>");
-document.write("<script language=\"javascript\" src=\"..\/resources\/testharnessreport.js\"><\/script>");
+var head_src = document.head.innerHTML;
+if (head_src.search(/\/testharness.js\W/) === -1) {
```

Comment:
@zqzhang 
- Similar codes exist in both nfc_common.js and unitcommon.js. 
In these scripts, we will add testharness.js/testharnessreport.js scripts to test page if related script not exist. This way can avoid the repeated load of testharness.js/testharnessreport.js. The repeated load can lead to issue XWALK-3459.